### DISCUSSION TITLE
如果你管理着很多不同种类的文档报告，模版很多，我如何通过NLP 或者通过版面分析把它们按文档内容或按排版分类？

### DISCUSSION THREAD

**Discussion by WangShunzhiDQ** [PaddlePaddle/PaddleNLP#4856] [General]
如果你管理着很多不同种类的文档报告doc\docx\excel\ppt\pdf\jpg等等，模版很多，文档种类也很多，我如何通过NLP 或者通过版面分析把它们按文档内容或按排版分类？
目前的思路是统统把文档按页转为图片，然后通过paddlenlp标注、信息抽取，按类别训练模型。但需要时间实验与验证。
paddleNLP技术日趋成熟，但我一直没法找到更好的办法，希望得到高人指点，有Demo更好

**Comment by 1061302569** [comment]
同问
